### PR TITLE
Bump to v0.10.0

### PR DESCRIPTION
**Pull request by varsill**

### CODE REVIEW

**Review by mat-hek — APPROVED** [review]

**Comment by varsill** [comment]
I won't merge it as I need to add a commit that updates documentation nesting, app name etc.

File: lib/membrane_h264_plugin/h265_parser.ex [hunk 1]
```diff
@@ -155,6 +155,14 @@ defmodule Membrane.H265.Parser do
                 """
               ]
 
+  @typedoc """
+  Format of the H265 stream, if set to `:annexb` NALUs will be separated by
+  a start code (0x(00)000001) or if set to `:hvc1` or `:hev1` they will be
+  prefixed by their size.
+  """
+  @type stream_structure ::
+          :annexb | {codec_tag :: :hvc1 | :hev1, nalu_length_size :: pos_integer()}
```
Comment (mat-hek) [inline lib/membrane_h264_plugin/h265_parser.ex:164]: could this be used by the spec for the `output_stream_structure` option?

File: mix.exs [hunk 2]
```diff
@@ -79,8 +79,8 @@ defmodule Membrane.H264.Plugin.Mixfile do
       extras: ["README.md", "LICENSE"],
       formatters: ["html"],
       source_ref: "v#{@version}",
-      filter_modules: "Membrane\.H264\.Parser",
-      nest_modules_by_prefix: [Membrane.H264.Parser]
+      filter_modules: "Membrane\.H26(4|5)",
```
Comment (mat-hek) [inline mix.exs:82]: let's remove this

**Review by mat-hek — APPROVED** [review]

File: mix.exs (hunk 2 above)
Comment (varsill) [reply mix.exs]: done

File: lib/membrane_h264_plugin/h265_parser.ex (hunk 1 above)
Comment (varsill) [reply lib/membrane_h264_plugin/h265_parser.ex:163]: done here H.264 and H.265

**Pull request merged**